interpreter/cling/lib/Interpreter/IncrementalJIT.cpp

```diff
@@ -207,7 +207,8 @@ class Azog: public RTDyldMemoryManager {
 
   void deregisterEHFrames() override {
 #ifdef LLVM_ON_WIN32
-    platform::DeRegisterEHFrames(Addr, Size);
+    // platform::DeRegisterEHFrames(Addr, Size);
```

Comment:
@vgvassilev I don't understand the asymmetry between `registerEHFrames` and `deregisterEHFrames`. `MemoryManager` has args to both, `RuntimeDyld` has args to none, we have args to one and call it `override`? Could you have a look, please?